carah_app/lib/providers/articles_provider.dart

```diff
@@ -36,12 +38,51 @@ class ArticlesProvider extends ContentProvider<Article> {
   }
 
   @override
-  Future<void> fetchDataByCategory(String id) async {
+  fetchLightDataByCategory(String uuid) async {
+    var connectivityResult = await (Connectivity().checkConnectivity());
+    if (connectivityResult == ConnectivityResult.mobile ||
+        connectivityResult == ConnectivityResult.wifi) {
+      var questionsFromCMS = await http.post(
+        Uri.parse('$baseUrl/graphql'),
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body:
+            '''{"query":"        {\\r\\n          node(uuid: \\"$uuid\\") \\r\\n          {\\r\\n              children(filter: {\\r\\n    }   \\r\\n            ){\\r\\n                elements {\\r\\n                    displayName\\r\\n                    uuid\\r\\n                    schema {\\r\\n                        uuid\\r\\n                    }\\r\\n                    \\r\\n                }\\r\\n            }\\r\\n          }\\r\\n        }","variables":{}}''',
+      );
+      lightItems =
+          jsonDecode(utf8.decoder.convert(questionsFromCMS.bodyBytes))['data']
+                  ['node']['children']['elements']
+              .map<LightContent>((element) {
+        return LightContent.fromJson(element);
+      }).toList();
+      lightItems.removeWhere((element) =>
+          element.category ==
+          "066e4aa01dc14ad6a8951e789c719bf6"); //Remove all image folders
+    } else {
+      items = _offlineBox.get("articles_$uuid")?.cast<Article>();
+      lightItems.clear();
+      for(Article tmpItem in items) {
+        LightContent tmpLightContent = LightContent(uuid: tmpItem.uuid, title: tmpItem.title, content: "", category: tmpItem.category);
```

Comment:
Wenn du das im Konstruktor änderst, dann kannst du das content Attribut hier (und bei den 2 anderen Konstruktoraufrufen) weglassen
```suggestion
        LightContent tmpLightContent = LightContent(uuid: tmpItem.uuid, title: tmpItem.title, category: tmpItem.category);
```